apps/api/resources/api/View/HiddenFieldsShape.ttl

```diff
@@ -0,0 +1,25 @@
+PREFIX dash: <http://datashapes.org/dash#>
+PREFIX vcard: <http://www.w3.org/2006/vcard/ns#>
+PREFIX acl: <http://www.w3.org/ns/auth/acl#>
+PREFIX schema: <http://schema.org/>
+PREFIX sh: <http://www.w3.org/ns/shacl#>
+PREFIX view: <https://cube.link/view/>
+
+<>
+  a sh:NodeShape ;
+  sh:targetClass view:View ;
+  sh:property
+    [
+      sh:path schema:isBasedOn ;
+      sh:minCount 1 ;
+      sh:maxCount 1 ;
+      sh:nodeKind sh:IRI ;
+    ],
+    [
+      sh:path acl:owner ;
+      sh:minCount 1 ;
+      sh:maxCount 1 ;
+      sh:nodeKind sh:IRI ;
+      sh:class vcard:Individual ;
+    ] ;
```

Comment:
A separate shape to ensure that there is always a base metadata and owner when updating views

Reply:
From the specific requirements of ssz-view-builder, I just wonder if the granularity required each user owning their views, or a single `role` shared by many users, say 'create-views`, `delete-views` etc.   (I don't know)

Reply:
This is exactly to address the functionality of having every view associated with the "author", as required to display on the overview page

Reply:
How we manage access may be a topic in the future

Reply:
for sure!, 

then it looks good to me.

Reply:
I would not put too much detail in the vocab until after implementing access control. (if it's required). 
at this stage, perhaps `schema:author` has the advantage that it doesn't suggest access control.

Reply:
I've just discovered that acl:owner expects a foaf:Agent 

```
    acl:owner     a rdf:Property;
         :comment """The person or other agent which owns this.
    For example, the owner of a file in a filesystem.
    There is a sense of right to control.   Typically defaults to the agent who craeted
    something but can be changed.""";
         :label "owner"@en;
         :range foaf:Agent .
``

Reply:
> the advantage that it doesn't suggest access control.

makes sense. let me rework that